functionary/builder/utils.py

```diff
@@ -102,6 +100,8 @@ def build_package(build_id: UUID):
     Args:
         build_id: ID of the build being executed
     """
+    docker_client = docker.from_env()
```

Comment:
I moved this because it's previous placement meant that docker was required even for the non-builder processes.